index.html

```diff
@@ -40,6 +40,174 @@
+         <ol>
+          <li>Fearless</li>
+          <li>Folklore</li>
+          <li>Red - TV</li>
+          <li>Midnights</li>
+          <li>Speak Now - TV</li>
+          <li>Reputation</li>
+          <li>Evermore</li>
+          <li>Lover</li>
+          <li>1989</li>
+          <li>Taylor Swift</li>
+          </ol>
+       </section>
+
+       <section>
+        <h2>Streaming data</h2>
+        <table>
+          <caption>Album streams on Spotify as of 03/10/2023, prioritising the most streamed edition of the album</caption>
+          <thead>
+            <tr>
+             <th>Album Name</th>
+             <th>Streams</th>
+            </tr>
+          </thead>
+           <tbody>
+            <tr><td>Reputation Stadium Tour Surprise Song Playlist</td>
+              <td>8,489,577,517</td></tr>
+            <tr><td>Lover</td>
+              <td>7,565,214,013</td></tr>
+            <tr><td>1989 (Deluxe Edition)</td>
+              <td>6,935,579,334</td></tr>
+            <tr><td>Midnights (The Til Dawn Edition)</td>
+              <td>6,680,458,440</td></tr>
+            <tr><td>Reputation</td>
+              <td>6,246,888,537</td></tr>
+            <tr><td>folklore: the long pond studio sessions (from the Disney+ special) [deluxe edition]</td>
+             <td>6,048,057,222</td></tr>
+            <tr><td>Red (Taylor's Version)</td>
+             <td>3,839,257,665</td></tr>
+            <tr><td>evermore (deluxe version)</td>
+             <td>3,608,027,511</td></tr>
+            <tr><td>Red (Deluxe Edition)</td>
+             <td>3,156,044,438</td></tr>
+            <tr><td>Speak Now (Deluxe Package)</td>
+             <td>2,971,482,221</td></tr>
+            <tr><td>Fearless (Taylor's Version)</td>
+             <td>2,857,619,152</td></tr>
+            <tr><td>Fearless</td>
+             <td>2,350,859,783</td></tr>
+            <tr><td>Speak Now (Taylor's Version)</td>
+             <td>1,284,323,154</td></tr>
+            <tr><td>Taylor Swift (Big Machine Radio Release Special)</td>
+             <td>1,123,430,864</td></tr>
+           </tbody>
+           <tfoot>
+            <tr><td>Total</td><td>63,156,819,851</td></tr>
+           </tfoot>
+        </table>
+       </section>
+
+       <section>
+        <h2>Taylor swift Documentaries</h2>
+         <ol>
+          <li><p>Taylor Swift: Speak Now World Tour Live (2011). This documentary is purely a recording of one of her performances on the Speak Now World Tour.</p><img src="./images/speak-now.jpg" alt="Speak Now World Tour Poster" /></li>
+
+          <li><p>The 1989 Word Tour Live (2015). Another recording of her tour but this time it includes commentary. The commentary comes from   Swift and her friends and one of the highlights of this documentary is when she talks about all of the guests she brought out on stage during the tour's run including Mick Jagger and Alanis Morisette.</p><img src="./images/1989.jpg" alt="1989 Word Tour Poster" /></li>
```

Comment:
I'm going to challenge you on the ordered list here - are these the order that the documentaries were released? Is there a specific order here? Otherwise you might be better of with a `ul`